docs/contributing/Compiler Test Plan.md

```diff
@@ -2,7 +2,8 @@ This document provides guidance for thinking about language interactions and tes
 
 # General concerns:
 - Completeness of the specification as a guide for testing (is the spec complete enough to suggest what the compiler should do in each scenario?)
-- Other external documentation
+- *Ping* for new breaking changes and general ping for partner teams (Bill, Kathleen, Mads, IDE, Razor)
+- Help review external documentation
```

Comment:
To @jaredpar 's point, I might clarify that this is our docs, not all 3rd party publiciations:

```suggestion
- Help review [first party documentation](https://docs.microsoft.com/dotnet/csharp) changes.
```